### ISSUE TITLE
Select component popup shows behind other components

### ISSUE THREAD

**Issue by BernardoS** [marcosmoura/vue-material#26]
``` xml
<md-toolbar>
    <div class="md-title">My App</div>
    <div class="field-group">
      <md-input-container>
        <label for="movie">Movie</label>
        <md-select name="movie" id="movie">
          <md-option value="Fight Club">Fight Club</md-option>
          <md-option value="Godfather II">Godfather II</md-option>
          <md-option value="Godfather III">Godfather III</md-option>
          <md-option value="Godfather">Godfather</md-option>
        </md-select>
      </md-input-container>
    </div>
  </md-toolbar>
  <md-toolbar>
    <div class="md-title">My App</div>
    <div class="field-group">
      <md-input-container>
        <label for="movie2">Movie</label>
        <md-select name="movie2" id="movie2">
          <md-option value="Fight Club">Fight Club</md-option>
          <md-option value="Godfather II">Godfather II</md-option>
          <md-option value="Godfather III">Godfather III</md-option>
          <md-option value="Godfather">Godfather</md-option>
        </md-select>
      </md-input-container>
    </div>
  </md-toolbar>
```

the select with id `movie` is mounted behind the second toolbar.
That is actually expected since it was declared first, so to solve it i'd expected that moving the select to after the toolbar would solve it, but it doesn't. 
this happens:  `Missing input/select/textarea inside md-input-container`, since obviously i move it out of the input container element....

**Comment by marcosmoura** [comment]
Yeah. This can't happen. The ```md-select``` component needs to attach the content in a dom element inside ```body``` tag. If the parent element have a ```overflow: hidden``` this will happen too. I'm creating the ```md-menu``` component and ```md-select``` will inherit and use some features that will fix those problems. You can track the progress on this two other issues: #24 #19 

Thank you for reporting this problem.

**Comment by marcosmoura** [comment]
Already implemented and will be shipped on the very next release (v0.3.0)
